frontend/components/Layout.js

```diff
@@ -7,12 +7,20 @@ import Footer from './Footer'
 class Layout extends Component {
   render() {
     return (
-      <div>
+      <Fragment>
         <Head />
-        <Header />
-        {this.props.children}
-        <Footer />
-      </div>
+        <div
+          style={{
+            display: 'flex',
+            minHeight: '100vh',
+            flexDirection: 'column'
+          }}
+        >
+          <Header />
+          <div style={{ flex: '1 0 auto' }}>{this.props.children}</div>
```

Comment:
Have you tried using a class that materialize already has to do this e.g. container?

Reply:
If you mean:
```js
      <Fragment>
        <Head />
        <Header />
        <div class="container">{this.props.children}</div>
        <Footer />
      </Fragment>
```
... it doesn't seem to do much.
I got the preceding code from the [docs](https://materializecss.com/footer.html).